### ISSUE TITLE
Release New Version

### ISSUE THREAD

**Issue by hyunminch** [joshtemple/lkml#66]
Hi! First of all, thanks so much for the work here.

I've noticed that #63 was not included in the latest release -- and I was wondering if a new release with this could be done.

Thanks!

**Comment by joshtemple** [comment]
Hey @hyunminch, how are you installing lkml? If you're using pip, you can install the latest version even without a release.

To do this, you would run this command, which installs from the latest commit that includes the fix you want:
```
pip install git+https://github.com/joshtemple/lkml.git@cd0bc861c858fc9d252a116fe804dad42b8a4c85
```

**Comment by hyunminch** [comment]
Hi @joshtemple, thanks for the reply. Actually I'm not directly using lkml locally or anything, it's that the project I'm using (https://github.com/linkedin/datahub) depends on lkml when ingesting looker resources so I thought a release could come in handy.

**Comment by joshtemple** [comment]
Got it! In that case, I'll push out a new patch release today.

**Comment by joshtemple** [comment]
Hey @hyunminch, I've released version 1.1.2 with the fix from #63.

**Comment by hyunminch** [comment]
Thank you so much @joshtemple 👍 👍
